src/compatibility/fvSolution.cpp

```diff
@@ -120,9 +129,8 @@ void updateCriteria(NeoN::Dictionary& solverDict)
     // Set default values for relative residual norm and iteration count
     if (solverDict.contains("relTol"))
     {
-        NeoN::Dictionary& criteriaDict = solverDict.subDict("criteria");
-        criteriaDict.insert("relative_residual_norm", solverDict.get<NeoN::scalar>("relTol"));
-        solverDict.remove("relTol");
+        NeoN::Dictionary& d = solverDict.subDict("criteria");
+        d.insert("relative_residual_norm", safeRetrieveScalarAndRemove(solverDict, "relTol"));
```

Comment:
When inserting max iterations, the dict name is `criteriaDict`.
Better to make the dict name consistent when inserting different (key, value) pairs

```suggestion
        criteriaDict.insert("relative_residual_norm", safeRetrieveScalarAndRemove(solverDict, "relTol"));
```